### DISCUSSION TITLE
Rotate polar plot for 0 degrees as north

### DISCUSSION THREAD

**Discussion by stephenberry** [alandefreitas/matplotplusplus#125] [Q&A]
Is there a way to rotate polar plots so that 0 degrees is north (by default it is east)? Furthermore, can the rotation be flipped to a clockwise increase in angle? This would allow the creation of true compass plots.

**Comment by alandefreitas** [comment]
If I remember correctly, the only way to currently do that would be to shift the coordinates and relabel the ticks. But that's too much of a workaround. 

That's a nice feature request though. We are limited by gnuplot for now, but we are working on a new backend in which this kind of thing will be much easier to create.

**Reply by stephenberry** [reply]
Thanks for your reply. It's certainly better to focus on the long term solution of a new backend. If this can be considered when making polar plots in the future that would be great!

**Reply by syrenson** [reply]
@alandefreitas : I have run into the same need as the original poster. Specifically, I need to represent angles in a polar plot (polar histogram to be exact) with the convention of 0 degrees located at the positive y-axis and positive angles measured clockwise (placing 90 degrees at the positive x-axis).  Could you provide an example of how to implement your suggestion of shifting the coordinate system and relabeling the ticks to accomplish this? I attempted to make this work through use of the xticks/xlabels/rticks member functions of the axes_handle class but had no luck. Thank you for your time and consideration!
